plugins/alerts/frontend/public/templates/vue-main.html

```diff
@@ -49,9 +62,9 @@ <h2> {{i18n('alert.plugin-title')}}
                 </div>
             </template> 
             <template v-slot="scope">
-                <el-table-column fixed="left" width="100" label="">
+                <el-table-column fixed="left" width="108" label="">
                     <template v-slot="rowScope">
-                        <el-switch :value="rowScope.row.enabled"
+                        <el-switch :value="rowScope.row.enabled"  class="switch-column"
```

Comment:
instead of using a custom class of switch column.
you can do this - https://countly.slack.com/archives/C018XS3SA0G/p1631607559006400
in the el-table-column, set type to switch